docs/blog/rendered/torch.ipynb

```diff
@@ -0,0 +1,583 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "60be381b-30ae-41ab-81f0-751815e60559",
+   "metadata": {},
+   "source": [
+    "# Ibis on &#x1F525;: Supercharge Your Workflow with DuckDB and PyTorch\n",
+    "\n",
+    "In this blog post we show how to leverage ecosystem tools to build an end-to-end ML pipeline using Ibis, DuckDB and PyTorch.\n",
+    "\n",
+    "Check out the live stream of this notebook here: https://www.youtube.com/live/L4_deAdStKs?feature=share\n",
+    "\n",
+    "Let's get started!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "9b4a41cd-0ab7-4f53-9b44-4532e8a4d30b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ibis\n",
+    "import ibis.expr.datatypes as dt\n",
+    "\n",
+    "from ibis import _, selectors as s, udf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9e746630-3e01-465f-9ee1-06b061465266",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ibis.options.interactive = True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1cc4cd52-59de-4658-871d-536de98b2a8e",
+   "metadata": {},
+   "source": [
+    "## Define a Function to Clean Inputs\n",
+    "\n",
+    "Let's define a function to clean the data in a few different ways:\n",
+    "\n",
+    "- Remove outliers (Z-score based)\n",
+    "- Remove negative trip distances and negative fare amounts\n",
+    "- Cast inputs to `float32`, since that's what PyTorch wants\n",
+    "\n",
+    "We use a function here ito ensure that we can run the same code on the test data set before inference."
```

Comment:
```suggestion
    "We use a function here to ensure that we can run the same code on the test data set before prediction."
```

I'm not trying to restart the `inference` vs. `prediction` mess that always crops up, but given that the UDFs and model classes are all called "Predict", I thought we should be consistent.